rllib/offline/dataset_reader.py

```diff
@@ -20,6 +77,9 @@ class DatasetReader(InputReader):
             "input_config"={
                 "format": "json",
                 "path": "/tmp/sample_batches/",
+                # By default, parallelism=num_workers.
```

Comment:
Same comment as the PR before (we should do this in some follow-up PR), just for documentation purposes:
```
input_config:
    type: "DatasetReader",
    [c'tor kwarg1]: ...,
    [c'tpr kwarg2]: ...,
```

Reply:
sorry, I must have missed something.
can you tell me which PR it is?
I'd love to fix documentation for sure.
I actually need to put up PR to add documentation for DatasetReader once this PR is in.